pages/genre/genre.go

```diff
@@ -42,3 +45,22 @@ func containsLowerCase(array []string, search string) bool {
 
 	return false
 }
+
+func averageGenreScore(user *arn.User, animes []*arn.Anime) string {
+	if user == nil {
+		return ""
+	}
+
+	counter := 0.0
+	scores := 0.0
+
+	for _, anime := range animes {
+
+		if user.AnimeList().Contains(anime.ID) {
```

Comment:
`user.AnimeList()` is a DB query and, even though it is very fast thanks to [nano](https://github.com/aerogo/nano), it should still be cached in a local variable called `animeList`.